ueberdosis/tiptap
# Fix suggestion collaboration bug

## Please describe your changes

Fix a bug where the suggestion re-renders when someone is typing above the cursor while collaborating.
Here's a screencast explaining the bug and the code:
https://www.loom.com/share/bbb7e73feae44a8c8ebfcbae77ad3eb5?sid=8fd24818-0b3b-4cea-9a54-b7bcb017d682

## How did you accomplish your changes
Change when onStart and onExit are called in the suggestion extension, so that they won't be called if we are just moving the suggestion but it will remain active. 

## How have you tested your changes
Used the changed version of the suggestion extension in our local environment, here's how it looks after the changes:
https://www.loom.com/share/facfc0b58a7a496199f2be07a8306969

## How can we verify your changes
1. Create an editor with the Suggestion and Collaboration extensions
2. Open two editors, with one of them initiate the suggestion and with the other one type above the cursor where the suggestion is
3. The suggestion should not re-render


## Remarks

[add any additional remarks here]

## Checklist


## Related issues
